docs/nanocloud-api.yml

```diff
@@ -950,29 +1014,78 @@ paths:
         200:
           description: User removed from the team
 
-  /files{filename}:
+  /files:
     get:
       tags:
         - files
       summary:
-        Get all configuration variables
+        Get the files list in user storage
+      responses:
+        200:
+          description: Return the files list
+          schema:
+            type: object
+            properties:
+              data:
+                type: array
+                items:
+                  "$ref": "#/definitions/Files"
+              meta:
+                type: object
+                properties:
+                  storageSize:
+                    type: string
+        401:
+          description: Unauthorized when you are not logged in
+          schema:
+            type: object
+            properties:
+              errors:
+                type: array
+                items:
+                  "$ref": "#/definitions/Errors"
+
+  /files{machines}{path}:
+    get:
+      tags:
+        - files
+      summary:
+        Get the files list from virtual machine
       parameters:
         - in: path
-          name: filename
-          description: Name of the file
+          name: machines
+          description: Set at true
+          required: true
+          type: string
+          format: string
+        - in: path
+          name: path
+          description: Path to browse
           required: true
           type: string
           format: string
       responses:
         200:
-          description: Return the configuration
+          description: Return the files list
           schema:
             type: object
             properties:
               data:
                 type: array
                 items:
                   "$ref": "#/definitions/Files"
+        401:
+          description: Unauthorized when you are not logged in
+          schema:
+            type: object
+            properties:
+              errors:
+                type: array
+                items:
+                  "$ref": "#/definitions/Errors"
+        500:
```

Comment:
We should usually not describe internal server error if they are not predictible